### PR TITLE
Support box shadow

### PR DESCRIPTION
**Pull request by You-J**
update
1. Dropbox limited multi-support
2. Support for SpreadRadius
3. Code support that was only supported in flutter preview

Screen Shot 2021-11-09 at 8 45 41 PM

after
Screen Shot 2021-11-10 at 4 40 05 AM
_add spreadRadius in box-shadow)_

test link: http://localhost:6626/to-code?framework=flutter&preview=vanilla&design=https://www.figma.com/file/iypAHagtcSp3Osfo2a7EDz/engine?node-id=3793%3A3893

text link2(group shadow): http://localhost:6626/to-code?framework=flutter&preview=vanilla&design=https://www.figma.com/file/iypAHagtcSp3Osfo2a7EDz/engine?node-id=4022%3A3893

### CODE REVIEW

**Review by softmarshmallow — CHANGES_REQUESTED** [review]
ignoreme

File: docs/figma-box-shadow.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+offset-x | offset-y | blur-radius | spread-radius | color
+
+inset | offset-x | offset-y | color
+
+Any number of shadows, separated by commas
+
+```css
+box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, 0.2);
+```
```
Comment (softmarshmallow) [inline docs/figma-box-shadow.md:37]: Can be better

File: packages/builder-css-styles/box-shadow/index.ts [hunk 2]
```diff
@@ -2,12 +2,17 @@ import { BoxShadowManifest } from "@reflect-ui/core/lib/box-shadow";
 import { color } from "../color";
 import { px } from "../dimensions";
 
-export function boxshadow(bs: BoxShadowManifest): string {
-  if (!bs) {
+export function boxshadow(bs: BoxShadowManifest[]): string {
```
Comment (softmarshmallow) [inline packages/builder-css-styles/box-shadow/index.ts:5]:
```ts
function boxshadow (...bs: BoxShadowManifest[]){
    /// ...
}
```

File: packages/designto-token/token-container/index.ts [hunk 3]
```diff
@@ -4,13 +4,15 @@ import { tokenizeBackground } from "../token-background";
...
+import { BoxShadowManifest } from "@reflect-ui/core";
+
 function fromRectangle(rect: nodes.ReflectRectangleNode): core.Container {
   const container = new core.Container({
     key: keyFromNode(rect),
     width: rect.width,
     height: rect.height,
     borderRadius: rect.cornerRadius,
-    boxShadow: rect.primaryShadow,
+    boxShadow: rect.shadows as BoxShadowManifest[],
```
Comment (softmarshmallow) [inline packages/designto-token/token-container/index.ts:15]: `as BoxShadowManifest[]` - necessary?
if true, explain why

**Review by softmarshmallow — CHANGES_REQUESTED** [review]

File: packages/designto-token/token-container/index.ts (hunk 3 above)
Comment (You-J) [reply packages/designto-token/token-container/index.ts:15]: shadows return type is `ReadonlyArray<BoxShadowManifest>`.

A type conflict occurs.

`get shadows` root is `design-sdk/figma-reflect-node/base.node.ts`

File: packages/builder-css-styles/box-shadow/index.ts (hunk 2 above)
Comment (You-J) [reply packages/builder-css-styles/box-shadow/index.ts:5]: Why use Spread Operator to objectify bs?

File: docs/figma-box-shadow.md (hunk 1 above)
Comment (You-J) [reply docs/figma-box-shadow.md:37]: Is this comment a question?
If this is a question, I'll add more info.

**Review by softmarshmallow — APPROVED** [review]

**Pull request merged**